docs/user/build/install/jdk.rst

```diff
@@ -105,20 +112,17 @@ One technique is to set up a batch file similar to the following:
    
    Placing JAVA_HOME on the path before System32 shortcuts this annoying "feature".
 
-Java Extensions (Optional)
-^^^^^^^^^^^^^^^^^^^^^^^^^^
+Java Extensions (Pre Java 9)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. note::
-   Optional
-   
-   GeoTools is now able to build with just a normal Java Software Developers Kit.
+Our image processing library is capable of making use of native code (packaged up as Java extensions).
 
-You should be aware that several of our raster formats are capable of making use of native code (packaged up as Java extensions).
-
-Java Developers Kit:
-* Can make use of JAI and ImageIO if they have been installed into your JDK
-Java Runtime Environment:
-* Can make use of JAI and ImageIO if you have installed them into your JRE
+========================== ========== ==========
+Installation               JDK        JRE
+========================== ========== ==========
+JDK Installation           Available  Available
+JRE Installation                      Available
+========================== ========== ==========
```

Comment:
This is really unclear and seems to have the same values on both the X and Y axes. Did you mean to include JAI and ImageIO somewhere in this table?

Reply:
added it as a block to each section